internal/db/migrations/atlas.sum

```diff
@@ -1,9 +1,10 @@
-h1:WZ5wk5jdbf+kr0zpGRFQunArB/HB2vUo1SOvq2gl5Z4=
+h1:RgJS0crW+ucBERGFywvV6oMkp/cfjqacfKBCzINRYho=
 1570659451_init.up.sql h1:zyiKKl39RqMxuEhop5jeeiPTxPiSSq00Tn6u06gyNmk=
 1710442322_nullable_aip_id.up.sql h1:vL4eG5YELXr3k4ymhHuRD/R7KpNt3/DNRhH26t83x3A=
 20250207193001_rename_package_table.up.sql h1:d2RjfIturPoFYMMtFocrMvvjEXEqDXDdxQRttcknX/0=
 20250210184720_use_atlas.up.sql h1:HiF/qJPOzETqCJeN9EmOO9c3abLzl8gL1ondUSglKDA=
 20250318151452_rename_preservation_tables.up.sql h1:/yVkOH3vIbAI9DHsRrQoOKCppTeSFEM1Zqt9a8QwO+M=
 20250319035441_remove_sip_columns.up.sql h1:YvnMvV2N0369vT8x+CjxQTya/gJXcMrYZRyTgZ2nbx0=
 20250319035720_rename_workflow_column.up.sql h1:l77ApLKkj0DMmsAhEiF/6U+O3/0Jo0Z/ujEjRTIUEZw=
-20250402173826_remove_workflow_type.sql h1:C5XkUwDd8hubHA9DtKpdKERvAgu+IGaZRbC/1SsKJ3I=
+20250402173826_remove_workflow_type.up.sql h1:P0w5tngLlCsfyYm1hW8LqdxDgVAYWIf9QM7oTLu5mJA=
```

Comment:
Changing this migration name is going to break existing DBs migrated to that point. As I noted in the other PR, I think we should stop using `.up` in the migration names.

Reply:
The migrations without the `.up` are not applied.  The "remove workflow type" migration was not applied in my dev environment before the change.